DIRECTORY.md

```diff
@@ -6,6 +6,9 @@
   * [Rsacipher](https://github.com/TheAlgorithms/Go/blob/master/ciphers/RSAcipher.go)
   * [Xorcipher](https://github.com/TheAlgorithms/Go/blob/master/ciphers/xorCipher.go)
 
+## Conversions
```

Comment:
You don't need to update this file manually. Please remove this.

Reply:
Not sure how to remove file changes. will deleting the file or just removing the added lines work?